internal/common/common.go

```diff
@@ -595,3 +596,22 @@ func IsPlatformExternal(platform *models.Platform) bool {
 func IsPlatformTypeExternal(platformType models.PlatformType) bool {
 	return funk.Contains(GetExternalPlaformTypes(), platformType)
 }
+
```

Comment:
Do we have enough platform related stuff here to make it worth creating platform package?

Reply:
we have, it's a bit scattered everywhere. I think the best place for all related platform stuff should be in provider package, but the way it's currently designed makes import cycles. I think it would be great to refactor it afterwards. WDYT?

Reply:
sure, let's refactor it separately